unittests/rpc/BaseTests.cpp

```diff
@@ -440,3 +440,83 @@ TEST_F(RPCBaseTest, IssuerValidator)
     err = spec.validate(failingInput);
     ASSERT_FALSE(err);
 }
+
+TEST_F(RPCBaseTest, SubscibeStreamValidator)
+{
+    auto const spec = RpcSpec{{"streams", SubscibeStreamValidator}};
```

Comment:
I notice it's all `Subscibe` here instead of `Subscribe` - probably need to change everywhere (search + replace in the project).

Reply:
Good eyesight